generators/app/templates/src/styles/base/_variables.scss

```diff
@@ -19,12 +19,13 @@ $colors: (
 
 //---------------------------------------------------------------
 // Easing
+// Moved to util/_eases and can be accessed with a function
```

Comment:
I think I'd prefer just to kill this given how much crap is in this file. You cool with that?